provider/ec2/ebs.go

```diff
@@ -445,17 +445,22 @@ func (v *ebsVolumeSource) DescribeVolumes(volIds []string) ([]storage.DescribeVo
 
 // DestroyVolumes is specified on the storage.VolumeSource interface.
 func (v *ebsVolumeSource) DestroyVolumes(volIds []string) ([]error, error) {
-	return destroyVolumes(v.env.ec2, volIds), nil
+	return destroyVolumes(v.env.ec2, volIds, false), nil
 }
 
-func destroyVolumes(client *ec2.EC2, volIds []string) []error {
+// ReleaseVolumes is specified on the storage.VolumeSource interface.
+func (v *ebsVolumeSource) ReleaseVolumes(volIds []string) ([]error, error) {
+	return destroyVolumes(v.env.ec2, volIds, true), nil
+}
+
+func destroyVolumes(client *ec2.EC2, volIds []string, release bool) []error {
```

Comment:
I've looked at this a bit more closely, and there's enough differences (which I missed) that these should just be separate methods. e.g. release should error if delete-on-termination is set, or if there are attachments in the attaching/attached state.